action/lib/index.js

```diff
@@ -11,14 +11,14 @@ import workflows from './workflows.js'
 // sleep function
 const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
 
-export default async function ({ token, delay, timeout }) {
+export default async function ({ token, delay, timeout, sha }) {
   let timer = 0
 
   // init octokit
   const octokit = github.getOctokit(token)
 
   // extract sha
-  const { sha, ref, runId: run_id } = github.context
+  const { ctx_sha, ref, runId: run_id } = github.context
```

Comment:
this variable is never used

Reply:
I was afraid to remove it completely because the `runId: run_id` thing is something I haven't seen before :)